daemon/api_test.go

```diff
@@ -189,6 +203,21 @@ func (s *apiBaseSuite) TearDownTest(c *check.C) {
 	unsafeReadSnapInfo = unsafeReadSnapInfoImpl
 	ensureStateSoon = ensureStateSoonImpl
 	dirs.SetRootDir("")
+
+	// TODO: do something mockable
+	snapstateCoreInfo = snapstate.CoreInfo
```

Comment:
I removed the comment, as I don't remember what I meant by it exactly either.

I *think* it was about having something like `snapstate.MockStuff()` or similar.